### PR TITLE
[doc] `Benchmark.pm`: Scope variables

### PR DESCRIPTION
**Pull request by rwp0**
This enhances the module's documentation.

### CODE REVIEW

**Comment by rwp0** [comment]
Can this be merged now that I fixed the conflicts?

File: lib/Benchmark.pm [hunk 1]
```diff
@@ -494,7 +494,7 @@ init();
 
 sub BEGIN {
     if (eval 'require Time::HiRes') {
-	Time::HiRes->import('time');
+	import Time::HiRes qw(time);
```
Comment (haarg) [inline lib/Benchmark.pm]: This is indirect object syntax, which we are trying to move away from.

Comment (rwp0) [reply lib/Benchmark.pm]: @haarg I fixed that

File: lib/Benchmark.pm [hunk 2]
```diff
@@ -494,7 +494,7 @@ init();
 
 sub BEGIN {
     if (eval 'require Time::HiRes') {
-	Time::HiRes->import('time');
+	 Time::HiRes->import('time');
```
Comment (haarg) [inline lib/Benchmark.pm]: This now has an extraneous space.

Comment (rwp0) [reply lib/Benchmark.pm]: I see, I fixed that as well, though the inconsistent indentation is on the file-level.

**Comment by rwp0** [comment]
@haarg can we merge already if all good now since hasConflicts is removed

**Comment by rwp0** [comment]
Any updates yet?

**Review by mauke — APPROVED** [review]

**Review by book — APPROVED** [review]

**Pull request merged**